### ISSUE TITLE
TypeError for 'scoring_method' when learning a structure

### ISSUE THREAD

**Issue by salvador-moreno** [erdogant/bnlearn#31]
bnlearn throws a TypeError when running the example code to learn a structure.

Code:
```
import bnlearn as bn
df = bn.import_example()
model = bn.structure_learning.fit(df)
```

Output:
```
[bnlearn] >Computing best DAG using [hc]
[bnlearn] >Set scoring type at [bic]
TypeError: __init__() got an unexpected keyword argument 'scoring_method'
---------------------------------------------------------------------------
TypeError                                 Traceback (most recent call last)
<ipython-input-7-026730c0bfe5> in <module>
----> 1 model = bn.structure_learning.fit(df, methodtype='hc')

~/Workspace/bntest/venv/lib/python3.6/site-packages/bnlearn/structure_learning.py in fit(df, methodtype, scoretype, black_list, white_list, bw_list_method, max_indegree, tabu_length, epsilon, max_iter, root_node, verbose)
    182                                epsilon=config['epsilon'],
    183                                max_iter=config['max_iter'],
--> 184                                verbose=config['verbose'],
    185                                )
    186 

~/Workspace/bntest/venv/lib/python3.6/site-packages/bnlearn/structure_learning.py in _hillclimbsearch(df, scoretype, black_list, white_list, max_indegree, tabu_length, epsilon, max_iter, bw_list_method, verbose)
    353     scoring_method = _SetScoringType(df, scoretype, verbose=verbose)
    354     # Set search algorithm
--> 355     model = HillClimbSearch(df, scoring_method=scoring_method)
    356 
    357     # Compute best DAG

~/Workspace/bntest/venv/lib/python3.6/site-packages/pgmpy/estimators/HillClimbSearch.py in __init__(self, data, use_cache, **kwargs)
     55         self.use_cache = use_cache
     56 
---> 57         super(HillClimbSearch, self).__init__(data, **kwargs)
     58 
     59     def _legal_operations(

~/Workspace/bntest/venv/lib/python3.6/site-packages/pgmpy/estimators/base.py in __init__(self, data, independencies, **kwargs)
    292             self.variables = self.independencies.get_all_variables()
    293 
--> 294         super(StructureEstimator, self).__init__(data=data, **kwargs)
    295 
    296     def estimate(self):

TypeError: __init__() got an unexpected keyword argument 'scoring_method'
```

**Comment by erdogant** [comment]
I can not reproduce the error. Which version are you using?

```
import bnlearn as bn
print(bn.__version__)

```

**Comment by salvador-moreno** [comment]
I'm using version 0.3.20 as well. I'm not sure if both problems might be related, but I also had problems when trying to load data different from 'sprinkler' or 'titanic' (e.g. 'asia', 'andes' or 'water') when using the `import_example` function (output: `[bnlearn] >Error: Loading data not possible!`)

However, I can load the DAG files for the same examples ('asia,', 'andes', 'water') if I use  the `import_DAG` function. 

I'm running bnlearn in a virtual environment with python 3.6.9 64-bit. Could this be related to the virtualenv?

**Comment by GeoTrouvetout** [comment]
Similar error here. 
Python 3.8.5.
I am on WSL (Windows Subsystem for Linux). 
Seems to coming from pgmpy/estimators/base.py not expecting scoring_method.

`>>>import bnlearn as bn`
`>>>df = bn.import_example()`
> [bnlearn] >Downloading example dataset..
100% [................................................................................] 1049 / 1049[bnlearn] >Import dataset..

`>>>model = bn.structure_learning.fit(df)`
>[bnlearn] >Computing best DAG using [hc]
[bnlearn] >Set scoring type at [bic]
Traceback (most recent call last):
  File "<stdin>", line 1, in <module>
  File "/usr/local/lib/python3.8/dist-packages/bnlearn/structure_learning.py", line 175, in fit
    out = _hillclimbsearch(df,
  File "/usr/local/lib/python3.8/dist-packages/bnlearn/structure_learning.py", line 355, in _hillclimbsearch
    model = HillClimbSearch(df, scoring_method=scoring_method)
  File "/usr/local/lib/python3.8/dist-packages/pgmpy/estimators/HillClimbSearch.py", line 57, in __init__
    super(HillClimbSearch, self).__init__(data, **kwargs)
  File "/usr/local/lib/python3.8/dist-packages/pgmpy/estimators/base.py", line 294, in __init__
    super(StructureEstimator, self).__init__(data=data, **kwargs)
TypeError: __init__() got an unexpected keyword argument 'scoring_method'
`

**Comment by erdogant** [comment]
I was able to reproduce the error and the bugs should be fixed in the latest version (v0.3.21). It occurred due to a combination of python with pgmpy. Can you check it?

```
import bnlearn as bn
print(bn.__version__)

```
You can update with:

```
pip install -U bnlearn
```

**Comment by salvador-moreno** [comment]
Yes, it worked! Both problems were solved, thanks a lot
